<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NdiViewer</class>
 <widget class="QWidget" name="NdiViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>528</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3" columnstretch="3,5">
   <item row="0" column="0">
    <widget class="QGroupBox" name="grpMarkers">
     <property name="title">
      <string>Markers</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QTableWidget" name="tblMarkersView">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustIgnored</enum>
        </property>
        <property name="autoScroll">
         <bool>true</bool>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="alternatingRowColors">
         <bool>false</bool>
        </property>
        <property name="textElideMode">
         <enum>Qt::ElideRight</enum>
        </property>
        <property name="gridStyle">
         <enum>Qt::SolidLine</enum>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>30</number>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>110</number>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>30</number>
        </attribute>
        <column>
         <property name="text">
          <string>x</string>
         </property>
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/icon/res/coordinates.ico</normaloff>:/icon/res/coordinates.ico</iconset>
         </property>
        </column>
        <column>
         <property name="text">
          <string>y</string>
         </property>
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/icon/res/coordinates.ico</normalon>
          </iconset>
         </property>
        </column>
        <column>
         <property name="text">
          <string>z</string>
         </property>
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/icon/res/coordinates.ico</normaloff>:/icon/res/coordinates.ico</iconset>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="grpCalibration">
     <property name="title">
      <string>Calibration</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" rowstretch="5,1" columnstretch="1,3,0">
      <item row="0" column="0">
       <widget class="QListWidget" name="lstToolName">
        <property name="currentRow">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QTableWidget" name="tblToolMarkers">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustIgnored</enum>
        </property>
        <property name="autoScroll">
         <bool>true</bool>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="alternatingRowColors">
         <bool>false</bool>
        </property>
        <property name="textElideMode">
         <enum>Qt::ElideRight</enum>
        </property>
        <property name="gridStyle">
         <enum>Qt::SolidLine</enum>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>30</number>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>75</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>30</number>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>x</string>
         </property>
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/icon/res/coordinates.ico</normaloff>:/icon/res/coordinates.ico</iconset>
         </property>
        </column>
        <column>
         <property name="text">
          <string>y</string>
         </property>
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/icon/res/coordinates.ico</normaloff>:/icon/res/coordinates.ico</iconset>
         </property>
        </column>
        <column>
         <property name="text">
          <string>z</string>
         </property>
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/icon/res/coordinates.ico</normaloff>:/icon/res/coordinates.ico</iconset>
         </property>
        </column>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QGroupBox" name="grpFunctions">
        <property name="title">
         <string>Functions</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,2,5,2">
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>start</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblSteps">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Steps:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cmbSteps">
           <item>
            <property name="text">
             <string>1. Model</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2. Calibration Needle</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>3. Revise Matrix</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4. Bone Drill</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnExec">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Execute</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
